wvdecrypter/wvdecrypter.cpp

```diff
@@ -497,13 +497,17 @@ void WV_CencSingleSampleDecrypter::GetCapabilities(const uint8_t* key, uint32_t
 {
   caps = { 0, hdcp_version_, hdcp_limit_ };
 
-  if (session_.empty())
+  if (session_.empty()) {
+    LOG::LogF(SSDDEBUG, "Widevine capabilities: session empty");
```

Comment:
When you use LogF you print also the method name
so imo add also "Widevine capabilities:" is superfluos

```suggestion
    LOG::LogF(SSDDEBUG, "Session is empty");
```

(this and others below)

Reply:
Thanks, I didn't notice the prefix was added already.